types/node/perf_hooks.d.ts

```diff
@@ -3,6 +3,22 @@ declare module 'perf_hooks' {
 
     type EntryType = 'node' | 'mark' | 'measure' | 'gc' | 'function' | 'http2' | 'http';
 
+    interface NodeGCPerformanceDetail {
+        /**
+         * When `performanceEntry.entryType` is equal to 'gc', `the performance.kind` property identifies
+         * the type of garbage collection operation that occurred.
+         * See perf_hooks.constants for valid values.
+         */
+        readonly kind?: number;
```

Comment:
should we narrow the type to `perf_hooks.constants`?

Reply:
No since they are bitflags.